README.md

```diff
@@ -48,30 +48,30 @@ LOG_FORMAT=json
 LOG_LEVEL=info
 ```
 
-#### `LOGGER_ENABLED`
+### `LOGGER_ENABLED`
```

Comment:
These titles were nested because the `### Environment variables` already uses `###`.

Reply:
No problem, I've reverted most of the README changes.